workspace_tools/export/exporters.py

```diff
@@ -12,12 +12,99 @@
 from workspace_tools.toolchains import TOOLCHAIN_CLASSES
 from workspace_tools.targets import TARGET_MAP
 
+from project_generator.generate import Generator
+from project_generator.project import Project
+from project_generator.settings import ProjectSettings
+
 class OldLibrariesException(Exception): pass
 
-class Exporter():
+class Exporter(object):
     TEMPLATE_DIR = dirname(__file__)
     DOT_IN_RELATIVE_PATH = False
 
+    # mapping mbed to progen targets
+    PROGEN_TARGETS = {
```

Comment:
Makes sense to me, although is it possible that progen will not support a given target? If so, we should have some way of indicating that.

Or more ideally, each target is not supported by default and we specify which ones are.